scripts/resolve-index-migration.sh

```
@@ -0,0 +1,56 @@
+fi
+
+echo "Migration $MIGRATION_NAME is not recorded. Attempting to mark as applied..."
+
+cd prisma-local
+MIGRATION_FILE="migrations/$MIGRATION_NAME/migration.sql"
+if [ -f "$MIGRATION_FILE" ]; then
+    # Calculate checksum (Prisma uses base64-encoded SHA256)
+    CHECKSUM=$(cat "$MIGRATION_FILE" | openssl dgst -sha256 -binary | base64 | tr -d '\n' || echo "")
+    NOW=$(date -u +"%Y-%m-%d %H:%M:%S.000")
+    
+    echo "Calculated checksum: ${CHECKSUM:0:20}..."
```

Shellcheck (0.11.0)

[warning] In POSIX sh, string indexing is undefined.

(SC3057)